backend/cmd/headlamp.go

```diff
@@ -39,6 +36,10 @@ import (
 	"k8s.io/client-go/tools/portforward"
 	"k8s.io/client-go/transport/spdy"
 
+	"github.com/headlamp-k8s/headlamp/backend/pkg/cache"
```

Comment:
For my own interest, can you please explain why these import lines were moved?

Reply:
Developer tools caused, recovered